thread/src/components/ContactTabs.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Container, Header, Content, Button, Text, Tabs, Tab, ScrollableTab } from 'native-base';
+
+import Contacts from './Contacts';
+import RequestsReceived from './RequestsReceived';
+import RequestsSent from './RequestsSent';
+
+const ContactTabs = () => (
+  <Container>
+    <Tabs renderTabBar={()=> <ScrollableTab />}>
+      <Tab activeTabStyle={{backgroundColor:'#66bb6a'}} activeTextStyle={{color:'white'}} tabStyle={{backgroundColor:'#66bb6a'}} textStyle={{color: 'white'}} heading="Contacts" children={<Contacts />} />
```

Comment:
Please lets avoid duplicated css code, refactor `<Tab />` please to a function based component